modules/Bio/EnsEMBL/Analysis/Hive/Config/GenblastHomology.pm

```diff
@@ -0,0 +1,532 @@
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps                  => catdir($self->o('output_path'), 'genome_dumps'),
+    # This one is used by most analyses that run against a genome flatfile like exonerate, genblast etc. Has slice name style headers. Is softmasked
+    softmasked_genome_file        => catfile($self->o('genome_dumps'), $self->o('species_name').'_softmasked_toplevel.fa'),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+
+    'min_toplevel_slice_length'   => 250,
+
+    'homology_models_path'        => catdir($self->o('output_path'),'homology_models'),
+
+    ensembl_analysis_script           => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    remove_small_orf_script             => catfile($self->o('ensembl_analysis_script'), 'genebuild', 'remove_small_orf.pl'),
+  
+    # cutoffs for removing small_orf genes
+    'small_orf_cutoff' => '100',
+    'intron_cutoff' => '75',
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+    mysql_dump_options => '--max_allowed_packet=1000MB',
+
+########################
+# Executable paths
+########################
+
+    'blast_type' => 'ncbi', # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+    'exonerate_path'         => catfile($self->o('software_base_path'), 'opt', 'exonerate09', 'bin', 'exonerate'),
+    'uniprot_genblast_batch_size' => 15,
+    'uniprot_table_name'          => 'uniprot_sequences',
+    genewise_path => catfile($self->o('binary_base'), 'genewise'),
+    'genblast_path'     => catfile($self->o('binary_base'), 'genblast'),
+    'genblast_eval'     => $self->o('blast_type') eq 'wu' ? '1e-20' : '1e-1',
+    'genblast_cov'      => '0.5',
+    'genblast_pid'      => '30',
+    'genblast_max_rank' => '5',
+    'genblast_flag_small_introns' => 1,
+    'genblast_flag_subpar_models' => 0,
+
+
+##################################
+# Memory settings for the analyses
+##################################
+    'default_mem'          => '900',
+    'genblast_mem'         => '1900',
+    'genblast_retry_mem'   => '4900',
+
+    
+########################
+# db info
+########################
+
+    'genblast_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+
+    'genblast_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
```

Comment:
It would be better to have `genblast_nr_server` and `genblast_nr_port` to allow maximum flexibility. They would need to be added in the section above too

Reply:
non redundant genblast db server settings added

Reply:
It looks like the change was lost in my perltidy requests

Reply:
Rerun of perltidy as requested.

Reply:
Changes effected